debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/AbstractRecordsProducerTest.java

```diff
@@ -147,6 +162,21 @@
         );
     }
 
+    protected List<SchemaAndValueField> schemasAndValuesForNumericDecimalTypeWithoutDecimals() {
+        final Struct dvs = new Struct(VariableScaleDecimal.schema());
+        dvs.put("scale", 1).put("value", new BigDecimal("10.0").unscaledValue().toByteArray());
+        final Struct nvs = new Struct(VariableScaleDecimal.schema());
+        nvs.put("scale", 1).put("value", new BigDecimal("22.0").unscaledValue().toByteArray());
+        return Arrays.asList(
+                new SchemaAndValueField("d", Decimal.builder(2).optional().build(), new BigDecimal("1.00")),
+     // DBZ-351 new SchemaAndValueField("dzs", Decimal.builder(0).optional().build(), new BigDecimal("10")),
```

Comment:
Is this commented line a left-over (same below)?

Reply:
No, please check the Jira issue, we still pass numeric as double from protobug decoder

Reply:
Ah, ok. Just assigned that one to 0.8.0, it's about time this gets fixed.